functional-tests/src/test/groovy/pages/app/DocumentsPage.groovy

```diff
@@ -0,0 +1,12 @@
+package pages.app
+
+import pages.app.BaseAppPage
+
+class DocumentsPage extends BaseAppPage {
+  static at = { $("#documents-header") }
+
```

Comment:
Why does it matter where the page is stored in the site? Yes that is a more thorough check, but what is the benefit?

Reply:
URL checking is standard in Geb, and gives us more confidence that we arrived at the right place.

In the case of dynamic URLs, it is almost more important, because we need to guarantee that we are editing the correct project/document, etc.

Reply:
done